contracts/interface/IBucketEncode.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.0;
 
 import "../middle-layer/resource-mirror/storage/BucketStorage.sol";
 
-interface IBucketRlp {
+interface IBucketEncode {
     function decodeCmnCreateAckPackage(
```

Comment:
It make no sense that `decodeCmnxxx` is a function of `IBucketEncode`, I will suggest `IBucketCodec`

Reply:
fixed